java/api/src/main/java/io/ray/api/parallelactor/ParallelActorExecutor.java

```diff
@@ -0,0 +1,3 @@
+package io.ray.api.parallelactor;
+
+public class ParallelActorExecutor {}
```

Comment:
What's the use of exposing a base class in the api module?

Reply:
Removed